src/components/sidebar/helpers/generate-install-view-nav-items.ts

```diff
@@ -15,7 +15,8 @@ import { ProductData } from 'types/products'
  */
 export const generateInstallViewNavItems = (
 	product: ProductData,
-	menuItems?: MenuItem[]
+	menuItems?: MenuItem[],
+	isEnterpriseMode: boolean = false
```

Comment:
Just for my own learning. When we set a default value like this, does that also translate to this being optional?

Reply:
Yep!